Practice1/src/css/android.css

```diff
@@ -0,0 +1,73 @@
+.android {
+    display: flex;
+    justify-content: space-around;
+    background-color: var(--blue-light);
+}
+
+.android-img-component {
+    width: 50%;
+    text-align: center;
+}
+
+.android-img {
+    padding-top: 50px;
+}
+
+.android-component {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 50%;
+}
+
+.android-component-input {
+    color: var(--white);
+    font-size: 40px;
+    line-height: 46px;
+    width: 449px;
+    height: 94px;
+    font-weight: 400px;
+}
+
+.common-img {
+    display: flex;
+    padding-top: 20px;
+}
+
+.img-googleplay {
+    margin-right: 40px;
+    height: 55px;
+    width: 187px;
+}
+
+.img-appstore{
```

Comment:
Code style. Should add a space before `{`.

Reply:
Fixed